tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -262,6 +285,8 @@ void parse_seconds() {
 
     fail_parse("1.2 1.3", "%S %S", time_ms);
     fail_parse("1.2 2.2", "%S %S", time_ms);
+
+    test_lwg_3536();

```

Comment:
This call is being added to `parse_seconds()`, but LWG-3536 isn't specific to parsing `seconds`, and the test physically parses `minutes`.

I believe that it would be clearer to move this call into `test_parse()`, immediately above `parse_seconds();`, so that the order of the function calls matches the order of the function definitions. It also makes logical sense for `test_parse()` to begin by verifying the LWG-3536 behavior for failed parses, before testing the various types.